pkgs/by-name/ax/ax25ms/package.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, protobuf
+, pkg-config
+, grpc
+, libtool
+, which
+, automake
+, libax25
+}:
+
+stdenv.mkDerivation {
+  pname = "ax25ms";
+  version = "unstable-2024-04-28";
+
+  src = fetchFromGitHub {
+    owner = "ThomasHabets";
+    repo = "ax25ms";
+    rev = "c7d7213bb182e4b60f655c3f9f1bcb2b2440406b";
+    hash = "sha256-GljGJa44topJ6T0g5wuU8GTHLKzNmQqUl8/AR+pw2+I=";
+  };
+
+  buildInputs = [
+    protobuf
+    grpc
+    libax25
+  ];
+
+  nativeBuildInputs = [
+    which
+    pkg-config
+    autoconf
+    libtool
+    automake
+  ];
+
+  postInstall = ''
+    set +e
```

Comment:
What is the motivation to `set +e` in the `postInstall` phase here and toggle it at the end again?

Reply:
Because otherwise, if one of the paths doesn't exist, it will silently fail without erroring or failing the build. We want to know if one of these programs changes in the future, without silently failing to include it.

Reply:
I'm all for explicitly failing if things change, @MatthewCroughan, yet I'm uncertain how `set +e` is achieving this, please help me understand.

From my understanding the `postInstall` phase iterates over all programs in `$out/bin` and prefixes them with `ax25ms-`. How could one of the paths not exist and this fail if the paths are "queried" and build time? What (obvious thing) am I missing here? I'm genuinely curious 🙂